### DISCUSSION TITLE
Some weird problems

### DISCUSSION THREAD

**Discussion by DJDQfff** [unoplatform/uno#13766] [General]
Some weird problems 

I'm trying to  migrate an uwp app to uno. Some problems have been sovled ,
however, there are some weird problems I don't know how to try to solve.

*************

1. `NavigetionView.PaneDisplayMode="Auto"`

   using auto property can work  on WinUI, 
   but on Android ,its displaymode will be leftcompact then
   changes from to LeftCompact to LeftMinimal.
   (I didn't debug to check the property ,this is only  what I see).

2. I have a `FlipVIew` control to bind source to a `ObservableCollection<IArchiveEntry>` 
   (The IArchiveEntry is from SharpCompress nuget,used to open .zip file ). 

   For the selectionChanged event:
   ```csharp
       private async void FLIP_SelectionChanged (object sender , SelectionChangedEventArgs e)
       {
           var selectedEntry = FLIP.SelectedItem as IArchiveEntry;
           var container = FLIP.ContainerFromItem(selectedEntry);
           if (container is FlipViewItem item)
           {
                   // ...do works
           }
       }
   ```

        On WinUI , the `if (container is FlipViewItem item)` can work correctly , 
but on Android ,`item` will be null.

3.  I defined a simple ValueConverter to set  control's visibility. 
    In my migrating-app. it can't work on Android(WinUI is OK)
   
    >  If I use debug for the Converter, it sometimes can work.

******************
* All of these problems are on Andorid ,WinUI is OK.

* For the 1.st and 3.rd, them can correctly work in empty projects. 

sorry for  these problems are not seriously handled.I feel little tired

**Comment by jeromelaban** [comment]
Thanks for the reports! It's generally best to create individual discussions for each questions you have, this will avoid confusion between responses. If you can reopen each of them with more details (preferably with a repro sample), it will help a lot.
